apps/files/src/components/Collaborators/NewCollaborator.vue

```diff
@@ -190,6 +190,7 @@ export default {
   mounted () {
     // Ensure default role is not undefined
     this.selectedNewRole = this.defaultRole
+    this.toggleCollaboratorsEdit(false)
```

Comment:
Move it to the root component of the collaborators sidebar pls - `FileSharingSidebar.vue`

Reply:
@LukasHirt Done!

Reply:
Thanks! 👍